src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+{
+  "sodium.option_impact.low": "Низкое",
+  "sodium.option_impact.medium": "Среднее",
+  "sodium.option_impact.high": "Высокое",
+  "sodium.option_impact.extreme": "Огромное",
+  "sodium.option_impact.varies": "Разное",
+  "sodium.options.pages.quality": "Качество",
+  "sodium.options.pages.performance": "Производительность",
+  "sodium.options.pages.advanced": "Расширенные",
+  "sodium.options.view_distance.tooltip": "Дальность прорисовки определяет, как далеко будет видна местность. Меньшие значения соответствуют меньшей площади, что улучшает частоту кадров.",
+  "sodium.options.simulation_distance.tooltip": "Дальность симуляции определяет, как далеко местность и сущности будут загружаться и обрабатываться. Меньшие значения могут уменьшить нагрузку на внутренний сервер и увеличить частоту кадров.",
+  "sodium.options.brightness.tooltip": "Управляет яркостью (гаммой) игры.",
+  "sodium.options.gui_scale.tooltip": "Устанавливает коэффициент масштабирования для интерфейса. При выборе \"авто\" всегда будет использоваться наибольший размер.",
+  "sodium.options.fullscreen.tooltip": "Если включено, игра будет открыта во весь экран (если доступно).",
+  "sodium.options.v_sync.tooltip": "Если включена, частота кадров будет синхронизирована с частотой обновления монитора, обеспечивая более плавную игру за счет небольшой задержки ввода. Может понизить производительность слабых систем.",
+  "sodium.options.fps_limit.tooltip": "Ограничивает максимальное количество кадров в секунду. Это поможет уменьшить расход заряда батареи и нагрузку на систему в обеспечении многозадачности. При включённой вертикальной синхронизации этот параметр будет игнорироваться, если он выше частоты обновления вашего экрана.",
```

Появился вариант за счёт отклонения от исходного текста немного сократить длину перевода, при этом улучшив восприятие:

```suggestion
  "sodium.options.fps_limit.tooltip": "Ограничивает максимальное количество кадров в секунду. Это поможет уменьшить энергопотребление и нагрузку на систему в обеспечении многозадачности. При включённой вертикальной синхронизации этот параметр будет игнорироваться, если он выше частоты обновления вашего экрана.",
```
Что думаете?
Лично я очень не люблю отступать от исходных формулировок, хотя это тот случай, когда перевод точнее описывает ситуацию, чем исходная строка.

---

Мне нравится